### ISSUE TITLE
Based on one pre-trained model, can I just continue to finetune this model for just some of the classes? 

### ISSUE THREAD

**Issue by RAYRAYRAYRita** [NVIDIA/pix2pixHD#169]
Hello!
I'm a little confused.
I have trained the model on cityscapes.
And now I want to finetune just one class, like road or building according to a new dataset.
In this case, can I fix some layers or modify some weights or any other method and continue training to achieve this ?
Appreciate if everyone can give me some advice.

**Comment by vishwas1234567** [comment]
just add the optimizer for the model properly as they require the best in
the class for the application and the data set is also huge and we all do
have to a clean data there may glitches in it thus extract it properly thus
add by a new learning rate and the optimizer may be better if you go with
the sparse based.

**Comment by RAYRAYRAYRita** [comment]
@vishwas1234567 Thanks for your idea.
Could you please be more specific. Where should I put this optimizer?
My new dataset is very small with about hundreds of images. I just want to change the texture of one class from cityscapes to this new dataset and hope other classes could stay unchanged.

**Comment by vishwas1234567** [comment]
when you start the training code the parameter would be accurate and prior
do some calculations if you are working on PyTorch  and TensorFlow I can
help.

**Comment by vishwas1234567** [comment]
if possible do add the parameters and the hyper parameters have to be good
also study some papers in the prior
